src/components/keyStats.test.js

```diff
@@ -0,0 +1,79 @@
+import React from "react";
+import * as redux from "react-redux";
+import {
+  render,
+  cleanup,
+  screen,
+  prettyDOM,
+  waitForElement,
+  fireEvent,
+} from "@testing-library/react";
+import App from "../App";
+import { KeyStats } from "./keyStats";
+import { useQuoteSelector } from "./componentHooks/useCompanySelector";
+import { useStatSelector } from "./componentHooks/useTickerSelector";
+
+jest.mock("./componentHooks/useQuoteSelector", () => ({
+  useQuoteSelector: () => {
+    return {
+     previousClose: 100,
+     low: 50,
+     high: 200,
+     latestVolume: 100,
+     marketCap: 1000,
+     open: 75,
+     week52Low: 30,
+     week52High: 250,
+     avgTotalVolume: 150
+        }
+    },
+}));
+jest.mock("./componentHooks/useStatsSelector", () => ({
+  useStatsSelector: () => {
+    return {
+        peRatio: 10,
+        earningsPerShare: 1,
+        dividendYield: 0.5
+    }
+  },
+}));
+
+describe("Key Stats component", () => {
+  let app;
+
+  beforeEach(async () => {
+    const { container } = render(<KeyStats />);
+    app = container;
+    
+    await waitForElement(() => screen.getByText("Previous Close"));
```

Comment:
You might not need these waitForElement statements in any of your tests because they are unit tests, and thus have no asynchronous work happening. Not certain, but would probably be worth a check.

Reply:
Okay I will try getting rid of await.